application/src/main/java/org/togetherjava/tjbot/commands/utils/MessageUtils.java

```diff
@@ -142,12 +142,15 @@ public static String abbreviate(String text, int maxLength) {
     }
 
     /**
-     * Mentions a guild channel by its id
+     * Mentions a guild channel by its id. If the given channelId is unknown the formatted text will
+     * say `#deleted-channel` in discord.
```

Comment:
javadoc is html, not markdown. use `<i>#deleted-channel</i>` instead.

Reply:
done